examples~/quickstart/client/Program.cs

```diff
@@ -135,9 +137,28 @@ void OnConnect(DbConnection conn, Identity identity, string authToken)
     local_identity = identity;
     AuthToken.SaveToken(authToken);
 
-    conn.SubscriptionBuilder()
-        .OnApplied(OnSubscriptionApplied)
-        .Subscribe("SELECT * FROM user", "SELECT * FROM message");
+    var subscriptions = 0;
+    SubscriptionBuilder<EventContext>.Callback waitForSubscriptions = (EventContext ctx) =>
+    {
+        // Note: callbacks are always invoked on the main thread, so you don't need to
+        // worry about thread synchronization or anything like that.
+        subscriptions += 1;
+
+        if (subscriptions == 2)
+        {
+            OnSubscriptionApplied(ctx);
+        }
+    };
+
+    var userSubscription = conn.SubscriptionBuilder()
+        .OnApplied(waitForSubscriptions)
+        .Subscribe("SELECT * FROM user");
+    var messageSubscription = conn.SubscriptionBuilder()
+        .OnApplied(waitForSubscriptions)
+        .Subscribe("SELECT * FROM message");
+
+    //conn.SubscriptionBuilder().OnApplied(OnSubscriptionApplied).SubscribeToAllTables();
```

Comment:
Can we just leave a comment saying that this is how you subscribe to all tables.